templates/cassiopeia/html/layouts/chromes/default.php

```diff
@@ -21,34 +21,23 @@
 }
 
 $moduleTag              = $params->get('module_tag', 'div');
-$moduleAttribs          = [];
+$modulePos 		= $module->position;
```

Comment:
```suggestion
$modulePos              = $module->position;
```
See my previous review comments.

Reply:
See above